netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -985,6 +985,71 @@ net_ebpf_ext_connect_redirect_filter_change_notify(
     NET_EBPF_EXT_RETURN_NTSTATUS(status);
 }
 
+#define DECLARE_SOCK_ADDR_CLASSIFY_LOG_FUNCTION(family)                 \
+    static void _net_ebpf_ext_log_sock_addr_classify_v##family##(       \
+        _In_z_ const char* message,                                     \
+        uint64_t transport_endpoint_handle,                             \
+        _In_ const bpf_sock_addr_t* original_context,                   \
+        _In_opt_ const bpf_sock_addr_t* redirected_context,             \
+        uint32_t verdict)                                               \
```

Comment:
Seems like the `verdict` type should be an enum type not `uint32_t`

Reply:
I think the verdict not an enum in Linux also -- `BPF_SOCK_ADDR_VERDICT_PROCEED`, `BPF_SOCK_ADDR_VERDICT_REJECT`

Reply:
Are you updating this in this PR?

Reply:
The `verdict` is return value from the ebpf program. it is an int right now (and not changed in this PR). I think we can update it in a separate PR? Also, if we are planning to move to enums, we should ideally change it for all the program types.

Reply:
I would also prefer changing it for all program types at the same time. I'm ok with moving the 'enum' update to a separate PR.